de/privacy/current.md

```diff
@@ -245,19 +245,29 @@ Diese Webseite nutzt die Google Cloud Plattform, ein Cloud Computing Service der
 
 Die Datenschutzbestimmungen von Google findest Du [hier](https://policies.google.com/privacy?hl=de).
 
-#### Highrise
+#### Hubspot
 
-Wenn Du dich dazu entscheidest, einen Spendenbeitrag über unser Spendenformular zu leisten, werden die durch das Spendenformular erfasste Daten auf Highrise gespeichert. Highrise ist ein Customer-Relationship-Management Dienst welcher in den USA ansässig ist. Die gespeicherten Daten umfassen:
+Wenn Du mit uns Kontakt aufnimmst, oder dich dazu entscheidest, einen Spendenbeitrag über unser Spendenformular zu leisten, werden die durch das Spendenformular erfasste Daten auf Hubspot gespeichert. Hubspot ist ein Customer-Relationship-Management Dienst welcher in den USA ansässig ist. Die gespeicherten Daten umfassen:
 
 - Deine Kontaktdaten,
 - die Höhe und den Rhythmus Deiner Spende
 - seit wann Du spendest.
 
-Highrise nutzt diese Daten nicht und gibt sie grundsätzlich auch nicht weiter. Highrise gibt gespeicherte Daten nur weiter, (i) mit Einwilligung seitens Serlo.org, (ii) um gesetzlichen Verpflichtungen nachzukommen oder (iii) im Falle eines Verkaufs des Unternehmens, einer Fusion mit einem anderen Unternehmen oder ähnlichen geschäftlichen Interaktionen.
+Hubspot nutzt diese Daten nicht und gibt sie grundsätzlich auch nicht weiter. Hubspot gibt gespeicherte Daten nur weiter, (i) mit Einwilligung seitens Serlo.org, (ii) um gesetzlichen Verpflichtungen nachzukommen oder (iii) im Falle eines Verkaufs des Unternehmens, einer Fusion mit einem anderen Unternehmen oder ähnlichen geschäftlichen Interaktionen.
 
-Highrise ist unter dem EU-US Datenschutzabkommen zertifiziert und damit verpflichtet, den EU-Datenschutzvorgaben nachzukommen. Des weiteren hat Serlo.org mit Highrise ein Data-Processing Agreement abgeschlossen, wodurch sich Highrise verpflichtet, die Daten unserer Nutzer zu schützen und sie nicht weiterzugeben. Die Verarbeitung erfolgt auf Grundlage des Art. 6 Abs. 1 lit. a DSGVO (Einwilligung) und Art. 6 Abs. 1 lit f (Berechtigtes Interesse). Du kannst die erteilte Einwilligung zur Speicherung der Daten jederzeit widerrufen. Die Rechtmäßigkeit der bereits erfolgten Datenverarbeitungsvorgänge bleibt vom Widerruf unberührt.
+Die Verarbeitung erfolgt auf Grundlage des Art. 6 Abs. 1 lit. a DSGVO (Einwilligung) und Art. 6 Abs. 1 lit f (Berechtigtes Interesse). Du kannst die erteilte Einwilligung zur Speicherung der Daten jederzeit widerrufen. Die Rechtmäßigkeit der bereits erfolgten Datenverarbeitungsvorgänge bleibt vom Widerruf unberührt.
 
-Die Datenschutzerklärung von Highrise findest Du [hier](https://highrisehq.com/privacy/).
+Die Datenschutzerklärung von Hubspot findest Du [hier](https://legal.hubspot.com/de/privacy-policy).
+
+#### Planetscale
```

Comment:
It makes more sense that Planetscale is under the topic 'Verarbeitung der Nutzerdaten' after 'Simple Analytics'.